protocol/scripts/bridge_events/bridge_events.go

```diff
@@ -125,13 +131,57 @@ func main() {
 	bankBalances := make([]banktypes.Balance, 0)
 	sortedAddresses := lib.GetSortedKeys[sort.StringSlice](balances)
 	for _, address := range sortedAddresses {
-		newBalance := balances[address]
-		bankBalances = append(bankBalances, banktypes.Balance{
-			Address: address,
-			Coins:   sdk.NewCoins(sdk.NewCoin(denom, sdk.NewIntFromBigInt(newBalance))),
-		})
+		bankBalances = append(
+			bankBalances,
+			banktypes.Balance{
+				Address: address,
+				Coins:   sdk.NewCoins(sdk.NewCoin(denom, sdk.NewIntFromBigInt(balances[address]))),
+			},
+		)
+	}
+	bankGenesisJson := cdc.MustMarshalJSON(banktypes.NewGenesisState(
+		banktypes.Params{},
+		bankBalances,
+		sdk.NewCoins(),
+		[]banktypes.Metadata{},
+		[]banktypes.SendEnabled{},
+	))
+	fmt.Printf("\"bank.balances\": %s\n", extractFieldFromJson(bankGenesisJson, "balances"))
+
+	// Stop here if not verbose.
+	if !verbose {
+		return
+	}
+
+	// Print x/auth accounts information.
+	genesisAccounts := authtypes.GenesisAccounts{}
+	for i, address := range sortedAddresses {
+		var ba authtypes.AccountI = &authtypes.BaseAccount{
+			Address:       address,
+			PubKey:        nil,
+			AccountNumber: uint64(i),
```

Comment:
To confirm, is this the correct `AccountNumber` value to use? It seems like this ordering based on the account address (since we are iterating addresses sorted by the string)

Or does this account number not matter?